notebooks/215-text-image-superresolution/215-text-image-superresolution.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Text Image Super Resolution with OpenVINO\n",
+    "\n",
+    "Super Resolution is the process of enhancing the quality of an image by increasing the pixel count using deep learning. This notebook shows the Text Image Super Resolution (TISR) which takes just one low resolution image with blured text, and output a high resolution image which have clear text. We use a model called [text-image-super-resolution-0001](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/text-image-super-resolution-0001/README.md) which is available from the Open Model Zoo."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Preparation\n",
+    "\n",
+    "### First let's import the packages we need"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import cv2\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "import sys\n",
+    "sys.path.append(\"../utils\")\n",
+    "from notebook_utils import load_image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepare the model we need\n",
+    "\n",
+    "We use `omz_downloader` to download the model from open model zoo. The models in open model zoo are already converted to OpenVINO Intermediate Representation (IR), so there is no need to use `omz_converter`.  \n",
```

Comment:
Please include the link to the tools you are referred to. For example https://docs.openvino.ai/latest/omz_tools_downloader.html